### PR TITLE
Precompile assets for foreman_openscap in nightlies

### PR DESCRIPTION
**Pull request by xprazak2**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like you need to add the apipie docs to ```%files```

**Comment by ares** [comment]
green now, could someone please take a look?

**Comment by ekohl** [comment]
Same comment as https://github.com/theforeman/foreman-packaging/pull/1864/files/03310d5a6f26cfebc8eb21cfc66cb5f85272c000#diff-ece72b6a723db9ecf80ad7a1b0231da6

**Comment by ares** [comment]
I think release should be bumped to 2 since it's a spec change.

**Comment by xprazak2** [comment]
I bumped the release version

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 1]
```diff
@@ -83,11 +83,14 @@ mkdir -p %{buildroot}%{foreman_bundlerd_dir}
 %{gem_instdir}/db
 %{gem_instdir}/config
 %{gem_instdir}/locale
+%foreman_apipie_cache_plugin
+%foreman_apipie_cache_foreman
```
Comment (ekohl) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Isn't this only needed for foreman itself and not plugins?

Comment (xprazak2) [reply rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Both are needed, first packages the actual files while the other adds a [link from core](http://koji.katello.org/kojifiles/work/tasks/3491/43491/build.log) to these files.

Comment (ekohl) [reply rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Looks like you're right and other plugins do this too.

**Comment by dLobatog** [comment]
nightly - http://koji.katello.org/koji/buildinfo?buildID=18448
1.16 - http://koji.katello.org/koji/buildinfo?buildID=18449

**Pull request merged**